js/components/report/interactive-iframe.js

```diff
@@ -42,14 +42,14 @@ export default class InteractiveIframe extends PureComponent {
   }
 
   render() {
-    const { src, width, height } = this.props;
+    const { src, width, height, style } = this.props;
     return (
       // eslint-disable-next-line react/no-string-refs
       <iframe ref="iframe"
         src={src}
         width={width || "300px"}
         height={height || "300px"}
-        style={{border: "none", marginTop: "0.5em"}}
+        style={style || {border: "none", marginTop: "0.5em"}}
```

Comment:
Is there a scenario in which merging styles would make more sense than choosing one or the other?

Reply:
I debated between them, but the specific case was wanting to not use a default style (`marginTop: 0.5em`). So if we merged we'd have to pass in an additional clearing style (`marginTop: 0`), instead of just omitting it.

While in this case passing in an additional prop isn't much, I think it's additional confusing/source for errors if you pass in a bunch of props, and get additional props you didn't expect.